tests/metrics/collect_mem_consmd.sh

```diff
@@ -128,38 +140,100 @@ function start_collect()
 # It kills the collectl process.
 function stop_collect()
 {
+	echo "Stop collectl"
 	kill "$COLLECT_PID"
 }
 
-# Get the average about a "csv" file produced
+# It launches a instance of smem tool
+# in order to get metrics about proportional
+# set size.
+function start_pss_monitor()
+{
+	bash ./smem_monitor.sh $PROC &
+	SMEM_PID=$!
+}
+
+# It terms smem process
+function stop_pss_monitor()
+{
+	kill "$SMEM_PID"
+}
+
+# Get PSS memory metrics
+function get_pss_metrics()
+{
+	pss_sum=0
+	pss_count=0
+	samples=""
+
+	while read line; do
+		pss_mem=$(echo "$line" | sed 's/[A-Za-z]*//g')
+		res=$(echo "$pss_mem > 0" | bc )
+		if [ $res -eq 1 ];then
+			pss_sum=$(echo "$pss_sum + $pss_mem"| bc -l )
+			pss_count=$(( $pss_count + 1 ))
+			if [ ! -z "$samples" ];then
```

Comment:
Hi, thank you for the comments and regarding to this point, I know that is not necessary to use “if [ ] ... fi” however I think is easier to read the code using them, I like when a user does not need to be an expert  in a specific language (in this case bash)in order to understand a source code. For that reason I think is better to be clear on each line. What do you think?